flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/JSONGenerator.java

```diff
@@ -187,8 +186,6 @@ private void decorateNode(Integer vertexID, ObjectNode node) {
 			node.put(PACT, "Operator");
 		}
 
-		StreamOperator<?> operator = streamGraph.getStreamNode(vertexID).getOperator();
```

Comment:
Is this change relevant to the rest of the PR? If not, could you extract changes in this file to a separate `[hotfix]` commit?

Reply:
Yes, new pr: https://github.com/apache/flink/pull/8343